### DISCUSSION TITLE
error: the required flag `--tsa-host-key' was not specified - I am abit lost

### DISCUSSION THREAD

**Discussion by tommy-mcgee** [concourse/concourse#8654] [General]
I followed these steps here: 
https://concourse-ci.org/concourse-web.html - Starting to form pat 1
Then I worked my way through to the web configurations. 
When it comes time to run `concourse web` this error
`error: the required flag --tsa-host-key' was not specified`
I am not fully  understanding what I missing. Does anything standout what I am doing wrong.
I am running fedora 37 server 
Thank you very much in advance!

**Comment by stellar-scottreed** [comment]
You need to run "sh generate" within the keys/ directory to generate the necessary keys.

**Comment by taylorsilva** [comment]
I think you missed the steps in the previous section/page about generating all the keys concourse needs to run: https://concourse-ci.org/concourse-generate-key.html
